### DISCUSSION TITLE
How to save with original file names?

### DISCUSSION THREAD

**Discussion by JagenSprom** [ImageMagick/ImageMagick#5865] [Help]
I am looking to use the "fill" command on an entire folder's worth of images as part of creating a MUGEN character. Is there a method for saving the output files with their original names so as to keep from having to run an individual command for each image? I'm just trying to turn each file a full green color.

![image](https://user-images.githubusercontent.com/120099346/206430496-1679bb48-1305-40b6-a6e9-92083c468fd7.png)

**Comment by snibgo** [comment]
What version of IM, on what platform?

I suggest using "magick mogrify".

If you don't supply a "-path", the input will be overwritten. If you do supply a "-path", files will be written in that directory.

**Reply by JagenSprom** [reply]
ImageMagick 7.1 on Windows 11.

I tried a few different commands before falling asleep, but I only remember mogrify and -draw -fill, though I may have not been using them correctly. In some cases a bunch of .png~ files were generated, which I thought might have been files in the process of being generated, but then I watched for a bit and nothing happened with them.
